textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/GramBooleanQuery.java

```diff
@@ -18,10 +18,10 @@
 		OR
 	}
 	QueryOp operator;
-	List<String> operandList;
-	List<GramBooleanQuery> subQueryList;
+	Set<String> operandSet;
+	Set<GramBooleanQuery> subQuerySet;
 	
-	int gramLength;
+	static int gramLength;
```

Comment:
In the `RegexToGramQueryTranslator.translate()`, we can pass a gramLength as a parameter. Does it solve the problem?